location/build.gradle.kts

```diff
@@ -23,13 +23,14 @@ kotlin {
     sourceSets {
         commonMain {
             dependencies {
-                implementation(project(":permissions", ""))
-                implementation(project(":logging", ""))
+                api(project(":permissions", ""))
+                api(project(":logging", ""))
+                api(project(":base", ""))
             }
         }
         commonTest {
             dependencies {
-                implementation(project(":test-utils", ""))
+                api(project(":test-utils", ""))
```

Comment:
don't forget about this one though (not sure why this was changes since this module is never exported anyway?)